app/code/Magento/Ui/view/base/web/js/lib/core/collection.js

```diff
@@ -224,7 +224,7 @@ define([
         regionHasElements: function (name) {
             var region = this.getRegion(name);
 
-            return region().length > 0;
+            return region().length > 0 && 'item' in region.first();
         },
```

Comment:
can you cover your new changes with jasmine. This is required for every new PR

Reply:
@mrtuvn Should it only cover the updated function?

Reply:
yes you need to cover your changes with tests as @VladimirZaets mentioned
jasmine or mftf. But i think better write jasmine if you can